connext_cmake_module/env_hook/connext.bat.in

```diff
@@ -38,6 +38,8 @@ if not exist "%_Connext_ENV_SETUP_SCRIPT%" (
 :: Call RTI's env setup script, piping stdout to nul, since they have echo on.
 call "%_Connext_ENV_SETUP_SCRIPT%" 1> nul
```

Comment:
This call looks like it renders the re-exporting of NDDSHOME below unecessary.

Reply:
that;'s correct, but this way is explicit